blog/en/blog/2023/05/24/weekly-report-0524.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Biweekly Report (May 08 - May 22)"
+keywords: 
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/05/21/Y0065svK_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5-%E5%85%AC%E4%BC%97%E5%8F%B7%E5%A4%B4%E5%9B%BE-%E8%8B%B1%E6%96%87.png
+---
+
+> From 5.08 to 5.22, 33 contributors submitted 92 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+We have also sorted out some issues for newcomers to the community to participate in! If you are interested, don't miss it!
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/05/24/yYk0cpo5_20230524.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/05/24/YL1CWg4i_new-contributors%20%281%29.png)
+
```

Comment:
@Qizeng-api7 ,the number just does not make sense, I know some of the contributors on the list did not contribute in this period. Would you double-check and update if needed?

cc @SkyeYoung @SylviaBABY @lijing-21 @EstelleRao @tzssangglass

Reply:
![0a5afc4914e34a05c4f5dee2641e6bd](https://github.com/apache/apisix-website/assets/36651058/24fdbf4c-b219-4912-8a3e-e7bfe49a895f)

Reply:
Please remove my name(lijing-21) from the list of contributors, thx!

Reply:
After checking the list, I only reviewed a few PR last month and did not participate in other content. Secondly, @lunarwhite was a 2022 APISIX Open Source Summer Documentation Project student. He has not been contributing to the APISIX project for a long time, so please double-check everything

Reply:
I've checked the contributor list, you used the wrong time period 2022.05.08-2022.05.22, which should be 2023 @Qizeng-api7 
https://github.com/nic-chen/github-statistic/pull/19

Reply:
pls check this list, my name(tzssangglass) should also be removed

Reply:
Fixed in https://github.com/apache/apisix-website/pull/1596 and https://github.com/apache/apisix-website/pull/1597. @lijing-22 @SylviaBABY @tzssangglass @KID-G

Reply:
...amazing